### ISSUE TITLE
Allow extensions to be added but not removed on sync

### ISSUE THREAD

**Issue by Tg7z** [shanalikhan/code-settings-sync#373]
**Use case:** 
Team includes external contractors / members that also work on different projects.
I want to use Settings Sync to ensure my development team all has key extensions installed.
I do not want this to remove other extensions they have installed as these may be critical for their other projects or integral to their personal productivity.

**Suggested config**
"sync.removeExtensions": false

**Comment by shanalikhan** [comment]
Thanks !

Will look into in future updates

**Comment by tysonmatanich** [comment]
This would resolve the issue I'm running into with workspace extensions constantly getting uninstalled.

**Comment by seanf** [comment]
@Tg7z Could you use this feature for your use case? https://code.visualstudio.com/docs/editor/extension-gallery#_workspace-recommended-extensions Although it sounds like you're after more than a "recommendation".

**Comment by shanalikhan** [comment]
If we set it to false in computer A , when you upload the settings, it will upload all the extensions in the GIST.

@tysonmatanich it would be great, if you can explain your scenario

**Comment by tysonmatanich** [comment]
I was having 2 issues. I switched to manual syncing which fixed part of my issue.

Fixed by manually syncing:
When multiple extensions needed added via a sync, often they don't all get installed. It then syncs the bad partially installed state, which then removes the extension from the other good machines. For example: sync results in installing A, B, and C but C fails then the state is overridden with only A and B (C missing). Then when opening on another machine with A, B, and C, sync results in C being removed. It seems that sometimes it takes a VScode restart to install all the extensions but auto sync gets in a bad state.

Not fixed:
It would be nice to have the option of not syncing all extensions. I might use 1, 2, and 3 on my work machine but only want 1 and 3 on my personal machine.
